simplq/src/components/common/SidePanel/SidePanel.module.scss

```diff
@@ -69,12 +69,13 @@
   }
 }
 
-.side-panel-item {
+.side-panel-item-active {
```

Comment:
This is a nitty-gritty thing, but this should be the normal `side-panel-item` and the below dark variant should be `side-panel-item-selected`.

Rationale: Since this is a generic/common component, not specific to pause/resume alone